### ISSUE TITLE
Markdown preview does not show images with relative file path

### ISSUE THREAD

**Issue by RWeigelt** [Microsoft/vscode#803]
In version 0.10.2 (and earlier versions) images in markdown files referenced using a relative path are not shown in the preview. Tested on Windows 7 and Windows 10.

The developer tools show that the `src` attribute value of the `img` tag is wrong (`/d%3A/Temp/folder/example.jpg` doesn't work in a browser, should be `file:///D:/Temp/folder/example.jpg` instead)
![screenshot](https://cloud.githubusercontent.com/assets/1882836/11460907/20a05cfe-96f7-11e5-8da6-dd97ac8c4379.png)

**Comment by bpasero** [comment]
@RWeigelt I cannot repro on my Windows, can you share this project with me please so that I have the exact same environment?

**Comment by RWeigelt** [comment]
@bpasero At first I was confused by the "share this *project*", then it dawned on me - that's exactly the problem:

* If I open the *folder* that contains the markdown file, the image will be displayed in the preview.
* If I open just the single markdown *file*, then the image will not be displayed.

Interesting: In both cases the same HTML is generated (e.g. `<img src="/d%3A/Temp/example.jpg" ...` for the image that's in the same folder as the markdown file).

The fact that - when opening a folder - the image file is "known" to vscode (it appears in the list of files of the folder) does not seem to be the vital point. I just tested to open a folder `D:\Temp\` with the following markdown file:
```md
![](example.jpg)

![](..\example2.jpg)
```
I copied the file `D:\Temp\example.jpg` to `D:\example2.jpg` (i.e. *outside* the folder opened in vscode) and the image is still displayed correctly in the preview.

So there's something happening in vscode when opening a folder that lets the preview know some kind of "base". The same needs to happen when opening a single file, too.
